apps/vr-tests-react-components/src/stories/MessageBar/MessageBar.stories.tsx

```diff
@@ -24,7 +24,7 @@ export default {
 const intents: MessageBarIntent[] = ['info', 'warning', 'error', 'success'];
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29482&commitId=d0350bdcfcdab08ecd3df6a51afa68869a11c30c&env=prod&clientType=FLUENTUI)


  <details><summary><b>MessageBar</b> 7 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|MessageBar.Intents.default.chromium.png|1490961| Changed|
|MessageBar.Intents - Dark Mode.default.chromium.png|1525955| Changed|
|MessageBar.Auto.default.chromium.png|1798531| Changed|
|MessageBar.Square.default.chromium.png|20| Changed|
|MessageBar.Intents - High Contrast.default.chromium.png|1554262| Changed|
|MessageBar.Intents - RTL.default.chromium.png|1490537| Changed|
|MessageBar.Multiline.default.chromium.png|1634117| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>